frontend/talentsearch/src/js/components/workPreferencesForm/WorkPreferencesForm.tsx

```diff
@@ -201,6 +201,30 @@ export const WorkPreferencesApi: React.FunctionComponent = () => {
       id,
       user: data,
     }).then((result) => {
+      if (result.data?.updateUserAsUser) {
+        if (result.data?.updateUserAsUser?.isProfileComplete) {
+          sessionStorage.setItem("currentProfileStatus", "Complete");
+        } else {
+          sessionStorage.setItem("currentProfileStatus", "InComplete");
+        }
```

Comment:
Is there any reason to store the new status in sessionStorage as a different value? You can compare the current value to the previous one without storing the current value.

Reply:
Yeah. Since we are checking in each form, Current value don't need to save as a sessionStorage. Initially , I tried to check it in profile page but realized that logic is wrong.( After getting un expected results)